packages/beacon-node/test/spec/general/bls.ts

```diff
@@ -80,7 +89,11 @@ function aggregate(input: string[]): string {
  */
 function aggregate_verify(input: {pubkeys: string[]; messages: string[]; signature: string}): boolean {
   const {pubkeys, messages, signature} = input;
-  return bls.verifyMultiple(pubkeys.map(fromHexString), messages.map(fromHexString), fromHexString(signature));
+  try {
+    return aggregateVerify(messages.map(fromHexString), pubkeys.map(fromHexString), fromHexString(signature));
+  } catch {
+    return false;
+  }
```

Comment:
@wemeetagain I added this try/catch for the spec tests but am updating the base lib to not throw for this so it better meets the spec.